packages/core/src/types/IImageCalibration.ts

```diff
@@ -0,0 +1,38 @@
+import CalibrationTypes from '../enums/CalibrationTypes';
+
+/**
+ * IImageCalibration is an object that stores information about the type
+ * of image calibration.
+ */
+export interface IImageCalibration {
+  /** The pixel spacing for the image, in mm between pixel centers */
```

Comment:
so I'm wondering whether there is a structure to this? If I understand this there are various calibration types, ERF, USER, PROJ, US. So is it true that for each of these we can/should have `row/columnPixelSpacing` and `scale`? or is it like `scale` is only valid when there is `PROJ`? and doesn't makes sense when it is `USER`.

Reply:
Scale is used for user projection as well as projected images.  The use of scale actually fixes the storing/loading of user calibrated data, and so is quite critical to the entire fix.  What I'd really like to have is:
1. Aspect ratio - the displayed aspect ratio, irrespective of pixel spacing
2. Imager pixel spacing - the raw imaging spacing
3. Scale - to account for user calibration changes as well as projected units
4. Type - to actually indicate the type, with one of them being "Uncalibrated" or something like that to indicate px units
5. Calibration function, to account for weird cases like US calibration (not fully supported initially)

That would then get rid of hasPixelSpacing entirely, and we would pass around a calibration object instead of pixel spacing etc.  The pixel spacing values would always be defined but would simply be in whatever units were specified by the type, which might be px.